src/lib/alpha.js

```diff
@@ -13,7 +13,7 @@ export const alpha = {
   'nn-NO': /^[A-ZÆØÅ]+$/i,
   'hu-HU': /^[A-ZÁÉÍÓÖŐÚÜŰ]+$/i,
   'pl-PL': /^[A-ZĄĆĘŚŁŃÓŻŹ]+$/i,
-  'pt-PT': /^[A-ZÃÁÀÂÇÉÊÍÕÓÔÚÜ]+$/i,
+  'pt-PT': /^[A-ZÃÁÀÂÄÇÉÊËÍÏÕÓÔÖÚÜ]+$/i,
```

Comment:
I will look deeper into the research above. But for now, we can add or edit tests for the addition/changes made on this part.

Reply:
All right. Since there aren't any tests regarding this locale, I'll write it.